snippets/kotlin.json

```diff
@@ -1,4 +1,11 @@
 {
+    "fun": {
```

Comment:
I understand what you are trying to do here, but I'm a little skeptical about this because what if I wanted a single line function with more than 1 parameter. There's no simple way to distinguish that from functions with braces. 🤔